sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
-
+## 1.0.1 (2021-05-26)
+- Dependency versions updated.
```

Comment:
Good to add the below header here and in other changelogs.

### Dependency Updates
- upgraded azure-core dependency to 1.16.0
- <Mention-other-dependencies that got updated>

Reply:
I guess thats going to be a lot of information and hard to maintain the practice overtime,  so lets keep it simple